libsplinter/src/admin/rest_api/resources/circuits.rs

```diff
@@ -12,23 +12,62 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::circuit::{AuthorizationType, DurabilityType, PersistenceType, Roster, RouteType};
+use std::collections::BTreeMap;
+
+use crate::circuit::{Circuit, Roster, ServiceDefinition};
 use crate::rest_api::paging::Paging;
 
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+pub(crate) struct ListCircuitsResponse {
+    pub data: Vec<CircuitResponse>,
+    pub paging: Paging,
+}
+
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
 pub(crate) struct CircuitResponse {
     pub id: String,
-    pub auth: AuthorizationType,
     pub members: Vec<String>,
-    pub roster: Roster,
-    pub persistence: PersistenceType,
-    pub durability: DurabilityType,
-    pub routes: RouteType,
-    pub circuit_management_type: String,
+    pub roster: Vec<ServiceResponse>,
+    pub management_type: String,
+}
+
+impl From<Circuit> for CircuitResponse {
+    fn from(circuit: Circuit) -> Self {
+        Self {
+            id: circuit.id().into(),
+            members: circuit.members().to_vec(),
+            roster: circuit.roster().into(),
+            management_type: circuit.circuit_management_type().into(),
+        }
+    }
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
-pub(crate) struct ListCircuitsResponse {
-    pub data: Vec<CircuitResponse>,
-    pub paging: Paging,
+pub(crate) struct ServiceResponse {
+    pub service_id: String,
+    pub service_type: String,
+    pub allowed_nodes: Vec<String>,
+    pub arguments: BTreeMap<String, String>,
```

Comment:
Curious what peoples' thoughts are about whether the `arguments` field should be included in this response.

Reply:
I personally think it should be included.